libs/blocks/quiz/quiz.js

```diff
@@ -88,8 +88,12 @@ const App = ({
       if (currentFlow && currentFlow.length) {
         setSelectedQuestion(questionList[currentFlow] || []);
       }
+      const lastQuestion = [...Object.keys(questionList)]?.pop();
```

Comment:
Adding `?.` to pop doesn't do anything, as you will always have an array. If you are unsure if questionList is defined, you would want to use `questionList || {}`